jest.config.js

```diff
@@ -4,4 +4,5 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  collectCoverageFrom: ['./src/**/*.{ts,tsx}'],
```

Comment:
Regularly, coverage collected only from **tested** files. It means that if you have 100 files, and only one is used in test you will have 100% coverage, despite other 99 files don't tested at all. This adds collecting coverage from files which were not imported in tests.